packages/xod-client/src/projectBrowser/containers/ProjectBrowser.jsx

```diff
@@ -245,8 +245,27 @@ class ProjectBrowser extends React.Component {
             hoverButtons={this.libraryPatchesHoveredButtons(path)}
           />
         )}
-      </PatchGroup>
-    ));
+      </PatchGroup>),
+    ]));
+    const installingLibsComponents = R.map(
+      ({ owner, name, version }) => ([
+        `${owner}/${name}`,
+        (<div
+          className="PatchGroup PatchGroup--installing library"
+        >
+          <span className="name">{owner}/{name}</span>
+          <span className="version">{version}</span>
+          <Icon name="spinner" spin />
+        </div>),
+      ]),
+      installingLibs
+    );
+
+    return R.compose(
+      R.map(R.nth(1)),
+      R.sortBy(R.nth(0)),
+      R.concat
+    )(libComponents, installingLibsComponents);
```

Comment:
Maybe use a `{name, component}` instead of tuple simulation for better readability?